src/app/features/tech-record/tech-record-routing.module.ts

```diff
@@ -48,6 +49,13 @@ const routes: Routes = [
     canActivate: [MsalGuard, CancelEditTechGuard],
     resolve: { load: TechRecordViewResolver }
   },
+  {
+    path: 'historic/unarchive-record',
```

Comment:
if we have the time looking at the router and working out /historic would be great